doc/release-notes.md

```diff
@@ -4,6 +4,13 @@ release-notes at release time)
 Notable changes
 ===============
 
-- Added a `z_converttex` RPC method to support conversion of transparent
-  p2pkh addresses to the ZIP 320 (TEX) format.
```

Comment:
PR needs to be rebased on `master` to fix this (the text has been removed there).